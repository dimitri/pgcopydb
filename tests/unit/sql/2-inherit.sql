  select *
    from public.child_model_1
order by id desc
   limit 2;
