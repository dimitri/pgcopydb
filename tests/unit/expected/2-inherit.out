-[ RECORD 1 ]----
id | 11000
a  | 1
b  | f
c  | [1, 2, 3, 4]
-[ RECORD 2 ]----
id | 10999
a  | 1
b  | f
c  | [1, 2, 3, 4]

